tests/modules/programs/vim-vint/basic-configuration.nix

```diff
@@ -0,0 +1,25 @@
+{ config, pkgs, lib, xdg, ... }:
+
+{
+  config.programs.vim-vint = {
+    enable = true;
+    settings = {
+      cmdargs = {
+        severity = "error";
+        color = true;
+        env = { neovim = true; };
+      };
+      policies = {
+        ProhibitEqualTildeOperator.enabled = false;
+        ProhibitUsingUndeclaredVariable.enabled = false;
+        ProhibitAbbreviationOption.enabled = false;
+        ProhibitImplicitScopeVariable.enabled = false;
+        ProhibitSetNoCompatible.enabled = false;
+      };
+    };
+    config.nmt.script = let configFile = xdg.configFile.".vintrc.yaml";
+    in "	  assertFileContent \\\n	  \"${configFile}\" \\\n	  ${
```

Comment:
The formatter screwed up the formatting here; should I leave it alone?

Reply:
I think it was not happy with the tabs. You should configure your editor to use spaces for indentation